content/blog/posts/join-our-team.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Join Our Team
+date: 2021-07-06
+authors:
+  - Eva Ruth Moravec
+hero: https://res.cloudinary.com/texas-justice-initiative/image/upload/v1625582253/TJJ-icon-only_imqijs.png
+---
+TJI functions thanks to talented [volunteers](https://texasjusticeinitiative.org/about) who work collaboratively to build and maintain our website and data tools. Our team meets weekly – remotely and, when volunteers in Austin are comfortable doing so again, in-person – and keep in touch during the week on Slack and email. We are lucky to have consistently had amazing volunteers, and we are looking to add a few people to the mix. 
+
+Our volunteers have a variety of skillsets, backgrounds and levels of experience, and regularly learn from and teach each other during our meetings and individually. We'd love to welcome folks who can work on projects with existing team members, and we're also looking for people to fill these new roles: 
```

Comment:
This is a tough sentence because of all the commas and such. I would break it down to two sentences with "Regularly learn..." being part of the second sentence

Reply:
I would also add a parenthetical about what some of the existing projects are